azure-pipelines.yml

```diff
@@ -18,66 +18,64 @@ jobs:
 
   # Install NuGet
   - task: NuGetToolInstaller@0
-    displayName: Install NuGet 5.6.0
+    displayName: Install NuGet 6.0
     inputs:
-      versionSpec: 5.6.0
+      versionSpec: 6.0.0
 
   # Install NerdBank GitVersioning
   - task: DotNetCoreCLI@2
     displayName: Install NBGV tool
     inputs:
       command: custom
       custom: tool
-      arguments: install --tool-path . nbgv
+      arguments: install -g nbgv
```

Comment:
Are there any differences between using the two different formats here?

Reply:
It's a global install. keeps the repo folder clean! I also read it helps in caching!!